### DISCUSSION TITLE
Add execution timestamp to Flows

### DISCUSSION THREAD

**Discussion by scouech** [directus/directus#16053] [Feature Requests]
At the moment there is no execution timestamp / date in the flow trigger payload. 
It would be good to add the timestamp to the $trigger payload so that we can use it elsewhere in the flow (i.e. adding it to a record etc)

✌️

**Comment by SeanDylanGoff** [comment]
Why can't you use a script operation to fetch this? Wouldn't you need a script operation anyways to use the resulting timestamp (except maybe for filtering where you have access to $NOW)?

**Reply by scouech** [reply]
That's the workaround at the moment, however, for even the purpose of recording properly flow execution times and for accuracy I thought it would be better to have it at the flow execution level rather than an extra step in the flow. 

In our case, precision isn't necessary a massive issue, but I would assume getting it as an extra step in the flow could impact certain use cases where accuracy needs to be higher? 

Another thing, is that the trigger outputs certain data, I would think the time should be part of the default payload, maybe it's overkill but if it's not too much work we might see some usefulness. :)

**Reply by rijkvanzanten** [reply]
> Another thing, is that the trigger outputs certain data, I would think the time should be part of the default payload

I agree! I also think there's something to say about having timestamps for operations as well (so you can see execution time in the logs)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
